Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN python3.7 -m pip install --upgrade \
 
 WORKDIR /home/$USERNAME/piro/
 
-# Set ip and 
-RUN find . -name 'app.py' | xargs sed -i "s/app.run_server(debug=True)/app.run_server(host='0.0.0.0', port=8888, debug=True)/g"
+# Set ip and port of Dash app for Docker
+RUN find . -name 'app.py' | xargs sed -i "s/app.run_server(debug=True)/app.run_server(host='0.0.0.0', port=8080, debug=True)/g"
 
 RUN python3.7 setup.py develop
```

Comment:
putting this line at the end of the Dockerfile should work too I think, if you didn't want to add docker-compose

`ENTRYPOINT python3.7 web/app.py`

Reply:
I wanted to keep a separate `docker run` command for those who wish to develop on `piro` within the docker container whereas `docker-compose` can be used by those who just want to spin up a local server quickly and run computations.